README.md

```diff
@@ -60,7 +63,29 @@ Then navigate to https://github.com/elaskavaia/bga-sharedcode/pulls, click "New
         ".vscode",
         ".git",
         ".DS_Store",
-        "misc/"
-    ]
+        "misc/",
+        "node_modules",
+        "package-lock.json"
+    ],
+    "syncOption": {
+        "skipCreate": false,
+        "delete": true 
+    }
 }
 ```
+- Sometimes you will have to run a manual push of your files and you can do that by using the command shortcut `Ctrl+Shift+P` and runing `SFTP: Sync Local -> Remote`
+- Install Run on Save extension (https://marketplace.visualstudio.com/items?itemName=pucelle.run-on-save)
+- Configure the Run on Save extention by clickin the Extensions tab in VSCode -> Gear icon on Run on Save -> Extension Settings -> Edit settings.json -> copy this into that file
```

Comment:
Can you add a note that these two steps are only needed if they want to edit SCSS? Most developers won't need to touch this.